modules/administration-guide/pages/customizing-openshift-che-consolelink.adoc

```diff
@@ -0,0 +1,41 @@
+:_content-type: PROCEDURE
+:description: Customizing OpenShift Eclipse Che ConsoleLink
+:keywords: administration guide, customizing, consolelink
+:navtitle: Customizing OpenShift Eclipse Che ConsoleLink
+:page-aliases: installation-guide:customizing-openshift-che-consolelink.adoc
+
+[id="customizing-openshift-che-consolelink"]
+= Customizing {ocp} {prod} ConsoleLink
+
+This procedure describes how to customize {prod} ConsoleLink.
```

Comment:
I would probably add more information about what the `ConsoleLink` is or cross-link it with the OpenShift docs